Mini/MiniCooperSE.json

```
@@ -0,0 +1,71 @@
+{
+  "init_commands": {
+    "command": [
+	"ATZ", 
+	"ATD", 
+	"ATE0", 
+	"ATS0", 
+	"ATAL",
+	"ATPBE101",
+	"ATSPB",
+	"ATBI",
+	"ATSH6F1",
+	"ATAT0",
+	"ATSTFF"
+	]
+  },
+  "data_commands": {
+    "command": [
+      "ATCRA607",
+      "ATCEA07",
+      "ATFCSH6F1",
+      "ATFCSD07300800",
+      "ATFCSM1",
```

```suggestion
	"ATSTFF",
	"ATCRA607",
        "ATCEA07",
        "ATFCSH6F1",
        "ATFCSD07300800",
        "ATFCSM1"
	]
  },
  "data_commands": {
    "command": [
```

Those are init commands as well and don't need to be sent again on each data query i think.

---

If you want data from another ecu i think you do need them in the data command section. If only the 607 ecu is needed then you are right. But for example is_charging is on another ECU i think. I wanted to start small with only 1 ecu